pytorch_pipeline/components/trainer/executor.py

```diff
@@ -1,12 +1,18 @@
+"""This module executues the training process and saves the model to checkpoint dir."""
+
+import os
+from argparse import Namespace
 import pytorch_lightning as pl
 import torch
-import os
-from pytorch_pipeline.components.trainer.generic_executor import GenericExecutor
 
 
-class Executor(GenericExecutor):
```

Comment:
@shrinath-suresh What is the reason removing the GenericExecutor. These will be needed for common elements s we add more components